tests/security/selinux/selinux_setup.pm

```diff
@@ -68,8 +69,15 @@ sub run {
     }
 
     # record the pkgs' version for reference
```

Comment:
@lilyeyes minor comments, could you please set the first character as CAP? in this file?

Reply:
will

Reply:
will